web/src/__snapshots__/AlertPane.test.tsx.snap

```diff
@@ -168,6 +206,36 @@ exports[`renders one container start error 2`] = `
         </code>
       </section>
     </li>
+    <li
```

Comment:
as we talked about in person, I think the fact that the snapshot is rendering two alerts for the container restart is a problem.